packages/core/src/tech/BaseTech.ts

```diff
@@ -338,7 +338,7 @@ export default class BaseTech extends EventEmitter {
     return this.video.volume;
   }
   set volume(newvol: number) {
-    this.video.volume = newvol;
+    this.video.volume = Math.round(newvol * 100) / 100;
```

Comment:
[opinion] instead of times 100, split by 100 - I would do `Number(newvol.toFixed(2))` or similar